postgresql/database.go

```diff
@@ -519,13 +523,15 @@ func (s *source) tableColumns(tableName string) ([]string, error) {
 		},
 	}
 
-	var rows *sql.Rows
+	var rows *sqlx.Rows
 	var err error
 
 	if rows, err = s.doQuery(stmt, s.schema.Name, tableName); err != nil {
 		return nil, err
 	}
 
+	defer rows.Close() // TODO: necessary? or leave it up to FetchRows?
```

Comment:
Since the `rows` variable was created in this block I think it also should be taken care of (closed) within the same block, just to be clear what happens with it. I think we could leave it here.